packages/api/app/Controllers/Http/AuthController.js

```diff
@@ -67,15 +66,15 @@ class AuthController {
 		user.status = 'verified';
 		await user.save();
 
-		const mailData = {
+		/* const mailData = {
```

Comment:
Is it necessary this confirmation mail now?

Reply:
I think it is not necessary